### PR TITLE
Add SR-IOV hardware networks xref to multiple networks

### PR DESCRIPTION
**Pull request by jboxman**
- https://bugzilla.redhat.com/show_bug.cgi?id=1820143

This was a regression.

### CODE REVIEW

File: networking/multiple_networks/understanding-multiple-networks.adoc [hunk 1]
```diff
@@ -63,3 +63,5 @@ networks in your cluster:
  similar to a macvlan-based additional network. Unlike a macvlan-based
  additional network, each Pod shares the same MAC address as the parent physical
  network interface.
+
+ * *SR-IOV*: xref:../../networking/hardware_networks/about-sriov.adoc#about-sriov[Creating a SR-IOV based additional network] allows Pods to attach to a virtual function (VF) interface on SR-IOV capable hardware on the host system.
```
Comment (bergerhoffer) [inline networking/multiple_networks/understanding-multiple-networks.adoc:67]: Is it "a SR-IOV" or "an SR-IOV"? I see it both ways when I click that link.

**Review by bergerhoffer — COMMENTED** [review]
Just one inconsistency question

**Comment by jboxman** [comment]
@bergerhoffer oops, thanks!

**Comment by jboxman** [comment]
@bergerhoffer, it looks like in the past [we've used](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/6/html/virtualization_host_configuration_and_guest_installation_guide/sect-virtualization_host_configuration_and_guest_installation_guide-sr_iov-how_sr_iov_libvirt_works) _an_ for RHEL 6.

That was my initial impulse.

**Pull request merged**